prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+"""
+CP1404/CP5632 Practical
+File reading, data processing and formatting output
+Data based on Wimbledon gentleman's singles champions
+"""
+
+FILENAME = "wimbledon.csv"
+INDEX_OF_COUNTRY = 1
+INDEX_OF_CHAMPION = 2
+
+
+def main():
+    """Read data from file and print Wimbledon champions with countries"""
+    records = get_records[FILENAME]
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
+def get_records(filename):
```

Comment:
This function reads the file, but forgets to return the results to be used in the main program.